catroid/src/org/catrobat/catroid/formulaeditor/SensorHandler.java

```diff
@@ -40,15 +40,23 @@
 	private static final String TAG = SensorHandler.class.getSimpleName();
 	private static SensorHandler instance = null;
 	private SensorManagerInterface sensorManager = null;
+	private Sensor linearAccelerationSensor = null;
 	private Sensor accelerometerSensor = null;
+	private Sensor magneticFieldSensor = null;
 	private Sensor rotationVectorSensor = null;
 	private float[] rotationMatrix = new float[16];
 	private float[] rotationVector = new float[3];
 	public static final float RADIAN_TO_DEGREE_CONST = 180f / (float) Math.PI;
 
-	private float linearAcceleartionX = 0f;
-	private float linearAcceleartionY = 0f;
-	private float linearAcceleartionZ = 0f;
+	private float[] accelerationXYZ = new float [3];
+	private float signAccelerationZ = 0f;
+	private float[] gravity = new float []{0f,0f,0f};
```

Comment:
`new float[] {0f, 0f, 0f};` is I think the correct formatting. Either way, run the formatter.